sections/04_study.md

```diff
@@ -27,8 +27,55 @@ dive into merits or weaknesses of individual methods.*
 
 ### Gene expression
 
-*Predicting gene expression levels and unsupervised approaches for learning
-from gene expression.  Those could be divided into separate sub-sections.*
+Gene expression measurements characterize the abundance of many thousands of
+RNA transcripts within a given organism, tissue, or cell. This characterization
+can represent the underlying state of the given system and can be used to study
+heterogeneity across samples as well as how the system reacts to perturbation.
+While gene expression measurements have been traditionally made by quantitative
+PCR, low throughput flourescense based methods, and microarray technologies, the
+field has shifted in recent years to primarily performing RNA sequencing to
+catalog total RNA. As such next generation sequencing technologies continue to
+fall in price and rise in throughput, studying gene expression through the lens
+of deep learning will increase in practicality and promise. 
```

Comment:
I think we always want more data - I didn't mean to imply that it is currently a limiting factor, although in some cases it definitely is. (E.g. studying gene expression of rare cancer types)

Instead, I meant to convey that increasing amounts of data make deep learning approaches more feasible (practicality) and that the pace of deep learning approaches applied to gene expression data is likely to grow leading to new discoveries and new methods (promise)

If you agree with that message, I will work on getting the point across more clearly.

Perhaps something like:

```
As such next generation sequencing technologies continue to fall in price and rise in throughput,
applying deep learning to study gene expression data is likely to make training more feasible. With increased modeling ability, deep learning approaches will grow in popularity and lead to novel biological insights.
```